### PR TITLE
upgrade: Small alignments with the design

### PR DESCRIPTION
**Pull request by skazi0**
There were some small differences between the implementation and
design wiki. Mainly texts which still need to be finalized.

### CODE REVIEW

**Comment by skazi0** [comment]
[(1) #UI Update UI - move to New Design (step 1 to 7)](https://trello.com/c/ZBzjl4Ay/218-1-ui-update-ui-move-to-new-design-step-1-to-7)

File: assets/app/features/upgrade/templates/openstack-services.jade [hunk 1]
```diff
@@ -4,7 +4,7 @@ p(translate='') upgrade.steps.openstack-services.description
 button.btn.btn-success.center-block(
     ng-click='upgradeOpenStackServicesVm.openStackServices.stopServices()',
     ng-disabled='upgradeOpenStackServicesVm.checks.services.running || \
-        upgradeOpenStackServicesVm.checks.backup.completed || upgradeOpenStackServicesVm.checks.backup.running',
```
Comment (Itxaka) [inline assets/app/features/upgrade/templates/openstack-services.jade:7]: This is not a design change right? This is a leftover from https://github.com/crowbar/crowbar-ui/pull/134/ ?

Comment (Itxaka) [reply assets/app/features/upgrade/templates/openstack-services.jade:7]: leftover = missing

Comment (skazi0) [reply assets/app/features/upgrade/templates/openstack-services.jade:7]: I found it while testing the behavior wrt the design spec so it's both actually. But you're right, it should have been included in #134.

**Review by Itxaka — APPROVED** [review]

**Pull request merged**